kf_lib_data_ingest/validation/values.py

```diff
@@ -0,0 +1,236 @@
+from dateutil.parser import parse as parsedate
+
+from kf_lib_data_ingest.common import constants
+from kf_lib_data_ingest.common.concept_schema import CONCEPT
+
+
+MIN_AGE_DAYS = 0
+# Max value chosen due to HIPAA de-identification standard
+# Using safe harbor guidelines
+# Equates to 90 years
+MAX_AGE_DAYS = 32872
+NULL_VALUES = {
+    constants.COMMON.NOT_REPORTED,
+    constants.COMMON.CANNOT_COLLECT,
+    constants.COMMON.NOT_APPLICABLE,
+    constants.COMMON.UNKNOWN,
+    constants.COMMON.NOT_AVAILABLE,
+}
+
+
+def try_wrap(description):
+    def wrapper(func):
+        def inner(x):
+            ret = True
+            if x is not None:
+                try:
+                    func(x)
+                except Exception:
+                    ret = False
+            return description, ret
+
+        return inner
+
+    return wrapper
+
+
+@try_wrap("Must be an integer x such that MIN_AGE_DAYS <= x <= MAX_AGE_DAYS")
```

Comment:
```suggestion
@try_wrap(f"Must be an integer x such that {MIN_AGE_DAYS} <= x <= {MAX_AGE_DAYS}")
```
I think you want this to be an f string so that the actual vals for those min/max appear

Reply:
yes oops